ec2/sc-ec2-linux-jumpcloud-development.yaml

```diff
@@ -352,7 +352,8 @@ Resources:
                 }
                 DEPARTMENT=$(extract_tag_value Department)
                 PROJECT=$(extract_tag_value Project)
-                OWNER_EMAIL=$(extract_tag_value OwnerEmail)
+                PROVISIONING_PRINCIPAL_ARN=$(extract_tag_value 'aws:servicecatalog:provisioningPrincipalArn')
+                OWNER_EMAIL=${!PROVISIONING_PRINCIPAL_ARN##*/}
```

Comment:
This is setting OWNER_EMAIL which goes into a file that can be source in different cfn-init steps to set values as environment vars. Since all that infrastructure is already in place, I don't need to update the other steps.

Reply:
Agreed- I think this is a good temporary fix. Maybe change the OwnerEmail tag to a UUID value in the future? This could be the Synapse ID of a user, which could also be the uid of a Jumpcloud user or the lookup value in the SSH pubkey table (when that happens).

Reply:
${!PROVISIONING_PRINCIPAL_ARN##*/} isn't going to give you the an email will it?  as is currently setup, won't it give you the synapse user id?

from..
https://docs.aws.amazon.com/cli/latest/reference/servicecatalog/search-provisioned-products.html

```
            "Tags": [
                {
                    "Value": "arn:aws:catalog:us-west-2:123456789012:product/prod-abcdfz3syn2rg",
                    "Key": "aws:servicecatalog:productArn"
                },
                {
                    "Value": "arn:aws:iam::123456789012:user/cliuser",
                    "Key": "aws:servicecatalog:provisioningPrincipalArn"
                },
```

Reply:
@zaro0508 Can we please fix this for current IAM users, which is the way we're supporting that people log in now, then we will do whatever retrofitting we need for the synapse solution when it is ready?

Reply:
sure